packages/multi-trait-rubric/src/main.jsx

```diff
@@ -7,20 +7,23 @@ import Scale from './scale';
 class Main extends React.Component {
   render() {
     const { model } = this.props;
-    const { halfScoring, scales, visible } = model || {};
+    const { halfScoring, scales, visible, pointLabels, description, standards } = model || {};
 
     if (!scales || !visible) {
       return <div/>;
     }
 
     return (
       <div>
-        {halfScoring ? <p>* Half-point or in-between scores are permitted under this rubric.</p> : null}
+        {halfScoring ? <p style={{ fontFamily: 'Cerebri Sans' }}>* Half-point or in-between scores are permitted under this rubric.</p> : null}
```

Comment:
I would try to move this fontFamily in the first-level (div, in this case) and check if it's children inherit it.
If it works this way, then we don't need to use fontFamily: 'Cerebri Sans' in the table, because it's inherited - in this way, we're consistent, and if any change is needed, we only have to change it in one place.

Reply:
Ok, I will move the fontFamily to the div.